src/algokit_utils/application_specification.py

```diff
@@ -62,7 +63,7 @@ class DefaultArgumentDict(TypedDict):
     data: int | str | bytes | MethodDict
 
 
-StateDict = TypedDict(  # noqa: UP013  # can't convert as "global" is a reserved keyword
+StateDict = TypedDict(  # can't convert as "global" is a reserved keyword
```

Comment:
```suggestion
StateDict = TypedDict(
```

Reply:
maybe `need to use function-form of TypedDict here since "global" is a reserved keyword`?